src/lib/requests/getLandingStats/getLandingStats.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { getLandingStats } from ".";
+
+describe("getLandingStats", () => {
+  it("should get the vewport by string", async (): Promise<void> => {
+    const stats = await getLandingStats();
+
+    expect(stats).toMatchObject({
+      usersCount: 27,
```

Comment:
Perhaps I'm missing something, but isn't this test gonna break once the actual stats change?

Reply:
We are using mocked data here so it will only break when mock data changes, which is fine.

Reply:
Okay, got it. Somehow I hadn't realized the stats are mocked as well.